seems like master is on go modules now and no longer uses glide

### Background

Docker build now fails with the following

Step 5/19 : RUN git clone https://github.com/btcsuite/btcd . &&  glide install &&  go install . ./cmd/...
 ---> Running in 477bfdfeb76c
Cloning into '.'...
[ERROR] Failed to find glide.yaml file in directory tree: Cannot resolve parent of /
The command '/bin/sh -c git clone https://github.com/btcsuite/btcd . &&  glide install &&  go install . ./cmd/...' returned a non-zero code: 2

### Steps to reproduce

docker-compose build

### Expected behaviour

successful build

### Actual behaviour

failed to build

--- a/docker/btcd/Dockerfile
+++ b/docker/btcd/Dockerfile
@@ -1,18 +1,15 @@
-FROM golang:1.10-alpine as builder
+FROM golang:1.11-alpine as builder
 
 MAINTAINER Olaoluwa Osuntokun <laolu@lightning.network>
 
 # Install build dependencies such as git and glide.
-RUN apk add --no-cache \
-    git \
-&&  go get -u github.com/Masterminds/glide
+RUN apk add --no-cache git gcc musl-dev
 
 WORKDIR $GOPATH/src/github.com/btcsuite/btcd
 
 # Grab and install the latest version of of btcd and all related dependencies.
-RUN git clone https://github.com/btcsuite/btcd . \
-&&  glide install \
-&&  go install . ./cmd/...
+RUN git clone https://github.com/btcsuite/btcd.git . \
+    &&  GO111MODULE=on go install -v . ./cmd/...
 
 # Start a new image
 FROM alpine as final

